### PR TITLE
Configuring

### PR DESCRIPTION
**Pull request by WojtekMs**

### CODE REVIEW

File: bowling/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+if (CMAKE_VERSION VERSION_LESS 2.8.11)
+  include_directories("${gtest_SOURCE_DIR}/include")
+endif()
+
+# Now simply link against gtest or gtest_main as needed. Eg
+enable_testing()
+add_executable(${PROJECT_NAME} main.cpp)
+add_executable(${PROJECT_NAME}-ut bowling_tests_gt.cpp)
+target_link_libraries(${PROJECT_NAME}-ut gtest_main)
+add_test(NAME bowlingTests COMMAND ${PROJECT_NAME}-ut)
```
Comment (Tomonike) [inline bowling/CMakeLists.txt:45]: We don't need a compilation flag ?

**Review by Tomonike — CHANGES_REQUESTED** [review]
Well done.

File: bowling/CMakeLists.txt (hunk 1 above)
Comment (WojtekMs) [reply bowling/CMakeLists.txt:45]: You are right, it should be done now :)

**Review by Tomonike — DISMISSED** [review]
Well done. :)

**Review by Tomonike — APPROVED** [review]
Ok

**Pull request merged**